src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,93 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Ui deals with interactions with the user.
+ */
+public class Ui {
+
+    private static final String BORDER = "    ============================================================";
+    private static final String INDENT = "     ";
+    private Scanner sc = new Scanner(System.in);
+
+    /**
+     * Shows the borders/outlines for Duke's messages.
+     */
+    public void showLine() {
+        System.out.println(BORDER);
+    }
+
+    /**
+     * Shows Duke's welcome message.
+     */
+    public void showWelcome() {
+        System.out.println(BORDER);
+        String content;
+        String logo = "      ______  ______  _    _  ______\n"
+                + "     |  ____||  ____|| |  | ||  __  |\n"
+                + "     | |____ | |     | |__| || |  | |\n"
+                + "     |  ____|| |     |  __  || |  | |\n"
+                + "     | |____ | |____ | |  | || |__| |\n"
+                + "     |______||______||_|  |_||______|\n";
+        content = logo + "\n" + INDENT + "Hello! I'm Echo\n" + INDENT + "What can I do for you?\n";
```

Comment:
Will it be better to use `System.lineSeparator()` rather than `\n`? I am unsure if `\n` can work on different OSes.